docs/reference/commands/disk.rst

```diff
@@ -64,6 +78,10 @@ Flags:
    as long as they are faster than the data device. WAL should take priority
    over DB if there isn't enough storage for both.
 
+   WAL and DB devices can only be used with data devices that reside on a
+   block device, not with loop files. Loop files do not support encryption.
```

Comment:
on a physical block device?

still not sure about "loop files". again i think we should settle on describing the device and not a type of file